### DISCUSSION TITLE
MS Teams notifications don't stay in NC

### DISCUSSION THREAD

**Discussion by VasilisManol** [ErikReider/SwayNotificationCenter#208] [Q&A] [answered]
Notifications coming from MS Teams chat don't stay in the notification center even though there are unread. Otherwise they appear as they should but they just disappear. Other notifications, of same urgency will appear and then stay in the NC until they are read or deleted.  
I have tried both Chrome and Firefox with the same result.
Is there a way to force all notifications to not disappear unless read ?

Thank you! 

SwayNotificationCenter: built from source: 23e406353a0c8199cff8290b5757a019243ff99a
SwayWM: 1.7
OS: Ubuntu 22.04.1

**Comment by ErikReider** [comment, marked as answer]
This is a MS Teams issue. Android Messages does the same thing. Nothing that I can do without breaking the Freedesktop Notification spec... Hope this clarified something at least :)

**Reply by VasilisManol** [reply]
Thanks for the reply! Can you point out why is this happening compared with other notifications? I would like to report it to Teams.

**Reply by georglauterbach** [reply]
Sorry to revive this, but this is still an issue. How can we report this to MS?
